instrumentation/kafka-streams/src/main/java/brave/kafka/streams/TracingPredicate.java

```diff
@@ -0,0 +1,43 @@
+package brave.kafka.streams;
+
+import brave.Span;
+import brave.Tracer;
+import org.apache.kafka.streams.KeyValue;
+import org.apache.kafka.streams.kstream.Predicate;
+
+class TracingPredicate<K, V> extends AbstractTracingTransformer<K, V, KeyValue<K, V>> {
```

Comment:
nit: instead of `TracingPredicate` consider `TracingFilterTransformer`, as predicate is just the condition object. 

Similar to https://github.com/apache/kafka/blob/trunk/streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamFilter.java#L39

Reply:
:+1: